### ISSUE TITLE
moveFromBuffer function unreachable

### ISSUE THREAD

**Issue by hberntsen** [richardgirges/express-fileupload#122]
I have an issue with the `mv` function:
```
{ [Error: EXDEV: cross-device link not permitted, rename '/tmp/tmp1550232515700' -> './images/BK00_small.png']
  errno: -18,
  code: 'EXDEV',
  syscall: 'rename',
  path: '/tmp/tmp1550232515700',
  dest: './images/BK00_small.png' }
```

So I looked at the code and found that there is a possibility to directly write from memory to a file: https://github.com/richardgirges/express-fileupload/blob/98e873c172b59190012a936e988ed1f64d3b10a4/lib/fileFactory.js#L20

In my experiments, even when `useTempFiles` is false, it still uses temporary files. When I remove the `&& !options.tempFilePath` part of the `if` statement I can successfully upload without temp files being used.

**Comment by RomanBurunkov** [comment]
Hi, issue with 'useTempFiles' was fixed in my PR #119.
Currently waiting review from @richardgirges.

**Comment by richardgirges** [comment]
v1.1.1-alpha.3 released on NPM: https://github.com/richardgirges/express-fileupload/releases/tag/v1.1.1-alpha.3
